index.js

```diff
@@ -16,41 +13,44 @@ var typedArrays = {
 	Uint16Array: true,
 	Uint32Array: true
 };
-
 var slice = String.prototype.slice;
-var toStrTags = {};
+var getpr = Object.getPrototypeOf;
```

Comment:
This method is part of ES5 and may not be available in every browser that has Typed Arrays (which shipped before ES5).